src/components/WindowsControls/index.tsx

```diff
@@ -24,8 +25,8 @@ export const WindowsControls = ({
   return (
     <div onMouseUp={onMouseUp} style={{ display: 'flex', ...style }}>
       <WindowsControl minimize whiteIcon={dark} onClick={onMinimize} />
-      <WindowsControl maximize isMaximized={isMaximized} whiteIcon={dark} onClick={onMaximize} />
-      <WindowsControl restore isMaximized={isMaximized} whiteIcon={dark} onClick={onMaximize} />
+      <WindowsControl maximize isMaximized={isMaximized == true} whiteIcon={dark} onClick={onMaximize} />
```

Comment:
I think it could be done like this:
```typescript 
<WindowsControl maximize={!isMaximized} restore={isMaximized} whiteIcon={dark} onClick={onMaximize} />
```